docs: Misc updates for HR (#14181)
unoplatform/uno: doc/articles/get-started-vs-2022.md

    - Press the `MyApp.Server` button to deploy the app
1. To debug for **iOS**:
    - Right click on the `MyApp.Mobile` project, select **Set as startup project**
        > [!NOTE]
        > For information about connecting Visual Studio to a Mac build host to build iOS apps, see [Pairing to a Mac for .NET iOS development](https://docs.microsoft.com/en-us/xamarin/ios/get-started/installation/windows/connecting-to-mac/).

    - In the "Debug toolbar" drop-down, select framework `net7.0-ios`:

      ![Visual Studio - "Debug toolbar" drop-down selecting the "net7.0-ios" framework](Assets/quick-start/net7-ios-debug.png)
      
    - Select:
      - An active device, if your IDE is connected to a macOS Host
      - A [local device using Hot Restart](https://learn.microsoft.com/en-us/xamarin/xamarin-forms/deploy-test/hot-restart), to debug your application without connecting to a mac
      

>         > [!NOTE] 
>         > If no iOS devices are available, a Visual Studio 17.7+ issue requires to unload/reload the project. Right click on the `.Mobile` project and select **Unload Project** then **Load project**.

Reviewer: ```suggestion
        > [!NOTE] 
        > If no iOS devices are available, a Visual Studio 17.7+ issue requires unloading/reloading the project. Right-click on the `.Mobile` project and select **Unload Project** then **Load project**.
```

>         > If no iOS devices are available, a Visual Studio 17.7+ issue requires unloading/reloading the project. Right-click on the `.Mobile` project and select **Unload Project** then **Load project**.

Reviewer: @jeromelaban do we need to share the link to the related Visual Studio 17.7+ issue ?

Author: It's not available yet, we'll update the docs when we do.

1. To debug the **Android** platform:
    - Right click on the `MyApp.Mobile` project, select **Set as startup project**
    - In the **Debug toolbar** drop down, select framework `net7.0-android`
    - Select an active device in the "Device" sub-menu

>         > [!NOTE] 
>         > If no android devices are available, a Visual Studio 17.7+ issue requires to unload/reload the project. Right click on the `.Mobile` project and select **Unload Project** then **Load project**.

Reviewer: ```suggestion
        > [!NOTE] 
        > If no android devices are available, a Visual Studio 17.7+ issue requires unloading/reloading the project. Right-click on the `.Mobile` project and select **Unload Project** then **Load project**.
```

>         > If no android devices are available, a Visual Studio 17.7+ issue requires unloading/reloading the project. Right-click on the `.Mobile` project and select **Unload Project** then **Load project**.

Reviewer: @jeromelaban  do we need to share the link to the related Visual Studio 17.7+ issue ?

You're all set, and don't forget to take a look at our [Hot Reload feature](xref:Uno.Features.HotReload)! You can now head to [our tutorials](getting-started-tutorial-1.md) on how to work on your Uno Platform app.

> [!NOTE] 

> > Debugging either the macOS and macCatalyst targets is not supported from Visual Studio on Windows. You can use [VS Code Remote SSH](xref:Uno.GetStarted.vscode) to enable this scenario.

Reviewer: ```suggestion
> Debugging either the macOS or macCatalyst targets is not supported by Visual Studio on Windows. You can use [VS Code Remote SSH](xref:Uno.GetStarted.vscode) to enable this scenario.
```
